ssr/src/page/notifs/mod.rs

```diff
@@ -0,0 +1,43 @@
+use leptos::*;
+
+use crate::{
+    component::canisters_prov::AuthCansProvider,
+    state::auth::account_connected_reader,
+    utils::{notifications::get_token_for_principal, profile::ProfileDetails},
+};
+
+#[component]
+fn NotifInnerComponent(details: ProfileDetails) -> impl IntoView {
+    let (_, _) = account_connected_reader();
+
+    // let on_token_click = move || {
+    //     let _ = create_local_resource(
+    //         || (),
+    //         move |_| async move { get_token_for_principal(details.principal.to_string()).await },
+    //     );
+    // };
```

Comment:
nit: Avoid keeping commented code that won't be used in the future

Reply:
will remove with next comm